functions/zoom-uploader.py

```diff
@@ -124,27 +130,58 @@ def s3_prefix(self):
     def zoom_series_id(self):
         return self.data['meeting_number']
 
+    def series_id_from_schedule(self):
+
+        dynamodb = boto3.resource('dynamodb')
+        table = dynamodb.Table(CLASS_SCHEDULE_TABLE)
```

Comment:
do you have something that populates this table yet?

Reply:
I do but it is a script in a jupyter notebook. If we decide on a standard json format we can add it to the project?

Reply:
:+1: